spec/features/medicaid_application_with_multiple_members_spec.rb

```diff
@@ -155,7 +155,7 @@
 
     on_page "Health Coverage Needs" do
       expect(page).to have_content(
-        "Tell us who is currently enrolled in a health insurance plan.",
+        I18n.t("medicaid.insurance_current_member.edit.title"),
```

Comment:
Using I18n.t in tests is more typing than you really need. Here’s how you can use t() instead of I18n.t() in all of your tests:
```
RSpec.configure do |config|
  config.include AbstractController::Translation
end
```
[From this article](https://robots.thoughtbot.com/foolproof-i18n-setup-in-rails)

Reply:
Do feature specs need to reference the translations file? It seems like it might be useful to make assertions about the final string output here.

Reply:
@bengolder it's nice to have them reference the translation because that means copy changes will not affect specs. I see what you're saying with it being nice to see the strings in test, but as long as we stick to a consistent naming scheme, finding the correct translations should be pretty easy.